models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql

```diff
@@ -104,9 +104,9 @@ known_solver_metadata (address, environment, name) as (
          ) as _
 )
 -- Combining the metadata with current activation status for final table
-select solver as address,
+select CAST(solver AS VARCHAR(100)) as address,
```

Comment:
nit: This is an address with a fixed predictable number of characters. Would it make sense to use that exact value (which is 42)

Reply:
Sure, I defaulted to 100 in the spirit of moving fast, but if we know it's 42 I can adapt this 👍